.github/workflows/ci-test.yaml

```diff
@@ -0,0 +1,132 @@
+name: Build and Run Tests
+
+on:
+#   # schedule:
+#     # - cron: '0 5 * * *'  #Runs  daily at 5 AM UTC
+  push:
+    branches:
+    - CV2-5258-run-linter-Check-web-github-actions
+
+
+env:
+  GITHUB_BRANCH: ${{ github.head_ref || github.ref_name }}
+  GITHUB_JOB_NAME: ${{ github.job }}
+  GITHUB_COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+  NGROK_AUTH: ${{ secrets.NGROK_AUTH }}
+  AWS_ENDPOINT: ${{ secrets.AWS_ENDPOINT }}  
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }} 
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }} 
+  AWS_REGION: eu-west-1
+  MAX_WEB_CURL: 3000
+  WEB_CURL_COUNT: 0
+
+jobs:
+  unit-tests:
+    if: > 
+        github.ref != 'refs/heads/develop' && github.ref != 'refs/heads/master' 
+        && !contains(github.event.head_commit.message, '[full ci]') && !contains(github.event.head_commit.message,
+        '[smoke tests]') && !contains(github.event.head_commit.message, '[similarity tests]')
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+        aws-region: eu-west-1
```

Comment:
Same question as above.

Reply:
This one does a different function to configure AWS access to the CI for commands involving docker push to ecr etc

Reply:
Ah, OK